packages/create-plugin/templates/common/src/README.md

```diff
@@ -1,5 +1,49 @@
-<!-- This README file is going to be the one displayed on the Grafana.com website for your plugin -->
+<!-- This README file is going to be the one displayed on the Grafana.com website for your plugin. Uncomment and replace the content here before publishing.
 
+Be sure to remove any remaining comments before publishing as these will be displayed on Grafana.com -->
 # {{ titleCase pluginName }}
 
 {{ sentenceCase pluginDescription }}
+
+<!-- To help maximise the impact of your README and improve usability for users, we propose the following loose structure:
+
+**BEFORE YOU BEGIN**
+- Ensure all links are absolute URLs so that they will work when the README is displayed within Grafana and Grafana.com
+- Be inspired ✨ 
+  - [grafana-polystat-panel](https://github.com/grafana/grafana-polystat-panel)
+  - [volkovlabs-variable-panel](https://github.com/volkovlabs/volkovlabs-variable-panel)
+
+**ADD SOME BADGES**
+
+Badges convey useful information at a glance for users whether in the Catalog or viewing the source code. You can use [Shields.io](https://shields.io/badges/dynamic-json-badge) together with the Grafana.com API 
+to create dynamic badges that update automatically when you publish a new version to the marketplace.
+
+- For the logo field use 'grafana'
+- Examples (label: query)
+  - Downloads: $.downloads
+  - Catalog Version: $.version
+  - Grafana Dependency: $.grafanaDependency
+  - Signature Type: $.versionSignatureType
```

Comment:
This is neat but I couldn't figure out what I actually needed to do to get the badge url. An example might make it much easier for people to make use of?

https://img.shields.io/badge/dynamic/json?logo=grafana&query=$.version&url=https://grafana.com/api/plugins/grafana-polystat-panel&label=Marketplace&prefix=v&color=F47A20